benefits/eligibility/views.py

```diff
@@ -118,22 +118,23 @@ def start(request):
     bank_card_item = dict(
```

Comment:
Same comment to use `viewmodels.MediaItem`

Reply:
Done in c14e5bb9733e51672d244934fef08cff6a1bc917